pyproject.toml

```diff
@@ -29,17 +30,14 @@ ray = { extras = ["default"], version = "^2.9.0" }
 cookiecutter = "^2.4.0"
 click = "^8.1.7"
 pygithub = "~=2.1.1"
-# extras: fastapi
-fastapi = { version = "~0.109.2", optional = true }
-uvicorn = { version = "^0.22", extras = ["standard"], optional = true }
-# extras: websocket_client
-websocket-client = { version = "^1.7.0", optional = true }
-websockets = { version = "^12.0", optional = true }
-# extras: rest_poller
-requests = { version = "^2.31.0", optional = true }
+
+# Optional Aineko Plugins
+aineko-plugins-nodes-fastapi-server = { version = "0.0.1", optional = true }
+aineko-plugins-nodes-http-poller = { version = "0.0.1", optional = true }
+aineko-plugins-nodes-websocket-client = { version = "0.0.1", optional = true }
```

Comment:
I wonder if we should get rid of this and make people install
`aineko_plugins['plugin_name']` because they would need to raise a PR in the aineko repo everytime they add a plugin.

Reply:
As discussed on Slack:
* created plugin-docs.aineko.dev
* removed extras from this pyproject
* removed the docs/plugins in favor of the dedicated page